pkgs/applications/networking/sniffers/sngrep/default.nix

```diff
@@ -0,0 +1,34 @@
+{stdenv, autoconf, automake, fetchFromGitHub, libpcap, ncurses, openssl, pcre}:
+
+stdenv.mkDerivation rec {
+  name = "sngrep-${version}";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "irontec";
+    repo = "sngrep";
+    rev = "v${version}";
+    sha256 = "0fj13pim5bfm3a2nr05apspraf29klpmcnhmycklfmrlncq5xqdf";
+  };
+
+  buildInputs = [
+    autoconf automake libpcap ncurses pcre openssl ncurses
+  ];
+
+  configureFlags = [
+    "--with-pcre"
+    "--enable-unicode"
+    "--enable-ipv6"
+    "--enable-eep"
+  ];
+
+  preConfigure = "sh bootstrap.sh";
+
+  meta = with stdenv.lib; {
+    description = "A tool for displaying SIP calls message flows from terminal";
+    homepage = https://github.com/irontec/sngrep;
```

Comment:
quotes round url

Reply:
I understood from the manual that URI's are a type in Nix and don't need quotes. In the nixpkgs repo I see url's with and without quotes. Is quotes around the homepage attribute the convention?

Reply:
Quotes are not needed. It is still in discussion whether it should be the convention. See https://github.com/NixOS/rfcs/pull/45

Reply:
Sorry about the very minimal review messages as well

Reply:
No problem! Thanks for the link, I didn't know I was running into a heated debate :) I picked quotes for now.